helm/charts/determined/values.yaml

```diff
@@ -233,15 +233,15 @@ checkpointStorage:
 # Distributed tasks with sizes that are not divisible by `maxSlotsPerPod` are never scheduled. If
 # you have a cluster of different size nodes (e.g., 4 and 8 GPUs per node), set `maxSlotsPerPod` to
 # the greatest common divisor of all the sizes (4, in that case).
-maxSlotsPerPod:
+maxSlotsPerPod: 1
```

Comment:
unneeded changes?

Reply:
i plan to revert all the helm changes before merging (but i have a note to make a `minikube-values.yaml` and merge it separately with a guide for local testing)